ui/core.js

```diff
@@ -126,6 +128,67 @@ $.fn.extend( {
 				$( this ).removeAttr( "id" );
 			}
 		} );
+	},
+
+	form: function() {
```

Comment:
You're replacing a function call with array indexing and property accesses. Either way, the performance isn't something to care about.

The reason I prefer the two line version is because it's clearer what's happening, and I prefer avoiding repeated `this[ 0 ]` where possible. Open to what others want to do.